src/ContainerIntegration.jsx

```diff
@@ -89,18 +87,18 @@ const ContainerEnv = ({ containerEnv, imageEnv }) => {
     return <List isPlain>{result}</List>;
 };
 
-const ContainerIntegration = ({ container, containerDetail, localImages }) => {
-    if (containerDetail === null || localImages === null) {
+const ContainerIntegration = ({ container, localImages }) => {
+    if (localImages === null) {
         return (
             <EmptyStatePanel title={_("Loading details...")} loading />
         );
     }
 
-    const ports = renderContainerPublishedPorts(container.Ports);
-    const volumes = renderContainerVolumes(containerDetail.Mounts);
+    const ports = renderContainerPublishedPorts(container.NetworkSettings.Ports);
+    const volumes = renderContainerVolumes(container.Mounts);
 
-    const image = localImages.filter(img => img.Id === container.ImageID)[0];
-    const env = <ContainerEnv containerEnv={containerDetail.Config.Env} imageEnv={image.Env || []} />;
+    const image = localImages.filter(img => img.Id === container.Image)[0];
+    const env = <ContainerEnv containerEnv={container.Config.Env} imageEnv={image.Env || []} />;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)